doc/release-process.md

```diff
@@ -1,83 +1,97 @@
+    git clone https://github.com/devrandom/gitian-builder.git
+    git clone https://github.com/bitcoin/bitcoin.git
+
+### Bitcoin maintainers/release engineers, update version in sources
+
+Update the following:
 
-###Bitcoin maintainers/release engineers, update (commit) version in sources
+- `configure.ac`: `_CLIENT_VERSION_MAJOR`, `_CLIENT_VERSION_MINOR`, `_CLIENT_VERSION_REVISION`. Don't forget to set `_CLIENT_VERSION_IS_RELEASE` to `true`.
+- `src/clientversion.h`: `CLIENT_VERSION_MAJOR`, `CLIENT_VERSION_MINOR`, `CLIENT_VERSION_REVISION`. Don't forget to set `CLIENT_VERSION_IS_RELEASE` to `true`.
+- `doc/README.md` and `doc/README_windows.txt`
+- `doc/Doxyfile`: `PROJECT_NUMBER` contains the full version
+- `contrib/gitian-descriptors/*.yml`: usually one'd want to do this on master after branching off the release - but be sure to at least do it before a new major release
 
-	pushd ./bitcoin
-	contrib/verifysfbinaries/verify.sh
-	configure.ac
-	doc/README*
-	doc/Doxyfile
-	contrib/gitian-descriptors/*.yml
-	src/clientversion.h (change CLIENT_VERSION_IS_RELEASE to true)
+Write release notes. git shortlog helps a lot, for example:
 
-	# tag version in git
+    git shortlog --no-merges v(current version, e.g. 0.7.2)..v(new version, e.g. 0.8.0)
 
-	git tag -s v(new version, e.g. 0.8.0)
+(or ping @wumpus on IRC, he has specific tooling to generate the list of merged pulls
+and sort them into categories based on labels)
 
-	# write release notes. git shortlog helps a lot, for example:
+Generate list of authors:
 
-	git shortlog --no-merges v(current version, e.g. 0.7.2)..v(new version, e.g. 0.8.0)
-	popd
+    git log --format='%aN' "$*" | sort -ui | sed -e 's/^/- /'
 
-* * *
+Tag version (or release candidate) in git
 
-###Setup and perform Gitian builds
+    git tag -s v(new version, e.g. 0.8.0)
 
- Setup Gitian descriptors:
+### Setup and perform Gitian builds
 
-	pushd ./bitcoin
-	export SIGNER=(your Gitian key, ie bluematt, sipa, etc)
-	export VERSION=(new version, e.g. 0.8.0)
-	git fetch
-	git checkout v${VERSION}
-	popd
+Setup Gitian descriptors:
 
-  Ensure your gitian.sigs are up-to-date if you wish to gverify your builds against other Gitian signatures.
+    pushd ./bitcoin
+    export SIGNER=(your Gitian key, ie bluematt, sipa, etc)
+    export VERSION=(new version, e.g. 0.8.0)
+    git fetch
+    git checkout v${VERSION}
+    popd
 
-	pushd ./gitian.sigs
-	git pull
-	popd
+Ensure your gitian.sigs are up-to-date if you wish to gverify your builds against other Gitian signatures.
 
-  Ensure gitian-builder is up-to-date to take advantage of new caching features (`e9741525c` or later is recommended).
+    pushd ./gitian.sigs
+    git pull
+    popd
 
-	pushd ./gitian-builder
-	git pull
+Ensure gitian-builder is up-to-date to take advantage of new caching features (`e9741525c` or later is recommended).
 
-###Fetch and create inputs: (first time, or when dependency versions change)
+    pushd ./gitian-builder
+    git pull
+    popd
```

Comment:
If you add this, you'd need to add `      pushd ./gitian-builder ` at the other places?